src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -125,63 +135,69 @@ Repeat these steps for each policy as needed.
 />
 
 <img
+  width="60%;"
+  class="center"
   title="A screenshot showing how to change your issue preferences on the alerts user interface."
   alt="A screenshot showing how to change your issue preferences on the alerts user interface."
   src={alertsIncidentPreferenceOptions}
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Alert conditions (Policies) > (select a policy):** The selected policy page shows how alerts rolls up issues for alert notifications and UI details. (Default is **By policy**). To choose a different option for this policy, click **Issue creation preference**.
+  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Alert conditions (policies) > (select a policy)**. The selected policy page shows how alerts rolls up issues for alert notifications and UI details. The default option is **One issue per policy**. To choose a different option for this policy, click **Issue creation preference**.
 </figcaption>
 
 ## One issue per policy (default) [#preference-policy]
 
+Choose the **One issue per policy** preference if your policy and its conditions have a focused scope, like monitoring one entity.
+
+If additional conditions open incidents after the issue opens, the incidents are added to the same issue. You will only see one issue on your **Issues & activity** page for the selected policy. Learn more about [your issue creation preferences](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#issue-creation).
+
 <img
-  title="A diagram showing how the By policy issue preference works"
-  alt="A diagram showing how the By policy issue preference works"
+  width="40%;"
+  class="center"
+  title="A diagram showing how the **One issue per policy** option works"
+  alt="A diagram showing how the **One issue per policy** option works"
   src={alertsByPolicy}
 />
 
 <figcaption>
-  When you select the default (**One issue per policy**) as your issue preference, it will group all condition incidents into the same issue.
+  When you select the default option **One issue per policy** as your issue preference, it will group all condition incidents into the same issue.
```

Comment:
I feel like this could be misunderstood: "it will group all condition incidents into the same issue"

So, each condition's incidents will get their own issue?

To make this clearer, we could add a word or two: "it will group all incidents opened by any condition on that policy into the same issue."

Reply:
Replaced!